graphql/resolvers/Comment.js

```diff
@@ -97,13 +106,22 @@ module.exports = {
     const testimonial = !anonymous && await pgdb.public.testimonials.findOne({userId: commenter.id})
     const profilePicture = testimonial && testimonial.image
 
+    // TODO maybe use a HMAC
+    const hash = crypto
+      .createHash('sha256')
+      .update(`${discussion.id}${commenter.id}${USER_ID_SALT}`)
+      .digest('hex')
+    const id = fakeUUID(hash)
```

Comment:
why is `fakeUUID` necessary? The way I understand GraphQL `ID` supports any string. If not necessary I'd skip it as it looses some bytes in the process – probably making collisions more likely.